command/sub8_missions/sub8/sub_singleton.py

```diff
@@ -312,6 +312,17 @@ def _run_look(self, spotings_req):
 
             yield self.sub.nh.sleep(.5)
 
+class PoseSequenceCommander(object):
```

Comment:
Are you sure we want command line passing? It seems like the list of poses on one line will get pretty unwieldy after 2 or 3. It might be a lot easier to just edit a python script.